db/transaction/transaction.go

```diff
@@ -59,8 +59,8 @@ type ResourceState struct {
 	Monitoring    string
 }
 
-//ListOptions specifies additional list related options.
-type ListOptions struct {
+//ViewOptions specifies additional list related options.
```

Comment:
it's not "list" anymore (same elsewhere). this is why I dislike comments in the code.